example/src/modules/AnimatedSignal/styles/styles.ts

```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
```

Comment:
@het-a-simform For consistency with other component no need to create styles directory. Other component has styles.ts file in component directory.

Reply:
Resolved, We need to keep this file in the styles folder I Updated the file name.